### ISSUE TITLE
selectedItemLabel performs differently in Shady DOM vs Shadow DOM

### ISSUE THREAD

**Issue by miztroh** [PolymerElements/paper-dropdown-menu#21]
In some cases, I need to provide descriptions of the options in my dropdown menus.  However, I don't want those descriptions to show up in the ``selectedItemLabel``.  To do this, I created a simple ``shadow-text`` element with a text property that renders the text into the local DOM of  that element.  This excludes the ``shadow-text`` content from the ``selectedItemLabel``.  In Polymer 0.5, this worked as expected.  It also works in Polymer 1.0+ in Shadow DOM mode.  Under Shady DOM, the ``shadow-text`` content is included in the ``selectedItemLabel``.  This is undesirable.  I've included my code for ``shadow-text`` below as well as some screenshots showing the differences in rendering.

```
<dom-module id="shadow-text">
	<template>
		<div>{{text}}</div>
	</template>
	<script>
		Polymer(
			{
				is: 'shadow-text',
				properties: {
					text: {
						type: String,
						value: ''
					}
				}
			}
		);
	</script>
</dom-module>
```

Here's how the label and description show in the dropdown:
![label and description](https://cloud.githubusercontent.com/assets/2515209/9283008/ed54e776-4295-11e5-876a-42cf7e668fc1.jpg)

Here's the selectedItemLabel rendering as expected under Shadow DOM:
![shadow dom](https://cloud.githubusercontent.com/assets/2515209/9283017/fcbddc22-4295-11e5-9045-01392173e1fd.jpg)

Here's the selectedItemLabel pulling in the text from ``shadow-text`` under Shady DOM
![shady dom](https://cloud.githubusercontent.com/assets/2515209/9283023/0aff1cc4-4296-11e5-8325-31f466b27818.jpg)

I wasn't sure if this is an issue with this one element or with the Polymer Shady DOM implementation.  Feel free to move this issue to a more relevant repo if necessary.

**Comment by cdata** [comment]
Yeah, this is an issue because the default value is derived from `textContent`. You can work around this by providing a `label` property on your selectable items. But, we should probably fix this so that the behavior is consistent.
